src/app/views/censusdata/create.html

```diff
@@ -1,27 +1,18 @@
 <div class="container">
-    <div class="well row" >
-        <h4>Census participants please be as complete as possible:</h4>
-        <ul>
-            <li>Review form guidelines prior to census.</li>
-            <li>Always provide "number of adults and juveniles" data for species.</li>
-            <li>Always observe birds from at least 50 yards away.</li>
-        </ul>
-    </div>
-                
+    <div ng-include="'/app/views/censusdata/progressBar.html'"></div>
+    {{censusForm.step = 1}}
+
     <form class="form-horizontal" role="form"> 
         <div class="form-group panel panel-primary">
-            <div class="panel-heading">Site Information: Section One<span style="float:right;">Report violations to 1-800-792-4263</span></div>
-            <div class="panel-body">              
+            <div class="panel-heading">Site Information<span style="float:right;">Report violations to 1-800-792-4263</span></div>
+            <div class="panel-body">
+                <!--{{ 2/2 }}    -->
             <div class="form-group">
-                <label class="control-label col-md-3" for="siteCode">Site Code:</label>
-                <div class="col-md-3">
-                <input type="text" class="form-control" id="siteCode" placeholder="Enter Site Code" ng-model="censusForm.siteCode">
-                </div>    
-            </div> 
-            <div class="form-group">
-                <label class="control-label col-md-3" for="siteName">Site Name:</label>
+                <label class="control-label col-md-3" for="location">Location:</label>
                 <div class="col-md-9">
-                <input type="text" class="form-control" id="siteName" placeholder="Enter Site Name" ng-model="censusForm.siteName">
+                 <select name="location"  class="form-control" id="location" ng-model="censusForm.locationId">
+                    <option ng-repeat="loc in list" value="{{loc.id}}">{{loc.siteCode}} | {{loc.siteName}} | {{loc.description}}</option>
```

Comment:
I have removed `description`, as it is redundant with `siteName`. Perhaps show use `{{loc.siteCode}} - {{loc.siteName}}` so that it comes out like "ABC123 - Some site in Texas".